Sources/Styles/Style+UIImageView.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Style+UIImageView.swift
+//  Pods-StyflerExample
+//
+//  Created by Valeriy Mikholapov on 30/03/2019.
+//
+
+public extension Style where Stylable: UIImageView {
+
+    static func image(_ image: ImageProvider) -> Style {
+        return .init(set: \.image, to: image.image)
+    }
+
+    static func highlightedImage(_ highlightedImage: ImageProvider) -> Style {
+        return .init(set: \.highlightedImage, to: highlightedImage.image)
+    }
+
+    static func animationImages(_ animationImages: [ImageProvider]) -> Style {
+        return .init(set: \.animationImages, to: animationImages.images)
+    }
+
+    static func highlightedAnimationImages(_ highlightedAnimationImages: [ImageProvider]) -> Style {
+        return .init(set: \.highlightedAnimationImages, to: highlightedAnimationImages.images)
+    }
+
+    static func animationDuration(_ animationDuration: TimeInterval) -> Style {
+        return .init(set: \.animationDuration, to: animationDuration)
+    }
+
+    static func animationRepeatCount(_ animationRepeatCount: Int) -> Style {
```

Comment:
Have you thought about grouping these two methods, because i think they are interconnected. With repeat count set as default parameter to real default value.

Something like this: 

```swift
static func animation(duration: TimeInterval, repeatCount: Int = <default>) -> Style
```

Reply:
Ok, I'll fix it.

Reply:
Done.